bonsai/Helpers/DateExt.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  DateExt.swift
+//  bonsai
+//
+//  Created by hoang on 12.12.2021.
+//
+
+import Foundation
+
+extension Date {
+    
+    private struct TransactionHeaderTitle {
+        static let today = "Today"
+        static let yesterday = "Yesterday"
+        
+        static func makeAnotherDate(date: Date) -> String {
+            let dateFormatter = DateFormatter()
+            dateFormatter.dateStyle = .long
+            return dateFormatter.string(from: date)
+        }
+    }
+    
+    static func dateMapper(date: Date) -> String {
```

Comment:
Cool! But I think `dateMapper` naming is not self-expressive
mapper from what to what?
When I suggested to implement this function, I called it mapper because it should map values to another values, but didn't mean it to be the name of the function :)

Also it could be instance property instead of static static method

In my opinion
`stringDescription: String`
or something like that could be good naming

Reply:
also get rid of that